.github/workflows/acceptance_tests_base.yml

```
@@ -213,6 +213,19 @@ jobs:
       - name: Run acceptance tests
         if: ${{ !inputs.skip_tests && needs.filters.outputs.require_acceptance_tests == 'true' }}
         run: ./testsuite/podman_runner/${{ inputs.tests }}
+      
+      - name: Upload Cucumber JSON Reports
+        uses: actions/upload-artifact@v4
+        with:
+          name: cucumber-json-reports
+          path: testsuite/results/**/*.json
```

Not fully sure about this, but it seems that the other steps uploading artifacts are following this relative path:

```suggestion
          path: ./testsuite/results/**/*.json
```